tests/util/test_package.py

```diff
@@ -19,9 +19,7 @@
 
 TEST_NEW_REQ = "pyhelloworld3==1.0.0"
 
-TEST_ZIP_REQ = "file://{}#{}".format(
-    os.path.join(RESOURCE_DIR, "pyhelloworld3.zip"), TEST_NEW_REQ
-)
+TEST_ZIP_REQ = f"file://{RESOURCE_DIR}/pyhelloworld3.zip/#{TEST_NEW_REQ}"
```

Comment:
I think you added an extra `/` here
```suggestion
TEST_ZIP_REQ = f"file://{RESOURCE_DIR}/pyhelloworld3.zip#{TEST_NEW_REQ}"
```

Reply:
Isn't the `/` between resource and file also extra?

Reply:
> Isn't the `/` between resource and file also extra?

I don't think so... previously it would have been added by `os.path.join`

Reply:
I would yes, I didn't look to carefully what it was doing 👍